packages/js-sdk/src/sandbox/index.ts

```diff
@@ -78,9 +78,9 @@ export class Sandbox extends SandboxConnection {
    * @internal
    * @access protected
    */
-  constructor(opts?: SandboxOpts) {
+  constructor(opts?: SandboxOpts, protected createCalled: boolean = false) {
```

Comment:
Add `protected`. Check also the protected constructor in `code-interpreter`.

Reply:
I created issue #375